### PR TITLE
Delete locations/Qs

### PR DESCRIPTION
**Pull request by TheBguy87**

### CODE REVIEW

File: helloCampusService.js [hunk 1]
```diff
@@ -18,11 +18,13 @@ router.use(express.json());
 
 router.get("/", readHelloMessage);
 router.get("/pointsOfInterest", readPointsOfInterest);
+router.delete("/pointOfInterest", deletePointOfInterest);
```
Comment (hangrydave) [inline helloCampusService.js:21]: this should be `/pointOfInterest/:id`

File: helloCampusService.js [hunk 2]
```diff
@@ -231,6 +233,28 @@ function createQuestion(req, res, next) {
         });
 }
 
+function deleteQuestion(req, res, next) {
+    db.one(
+"DELETE FROM Question WHERE ID = ${id}", req.params)
```
Comment (hangrydave) [inline helloCampusService.js:238]: I'd recommend this, same for the other one

```suggestion
function deleteQuestion(req, res, next) {
    db.one(
"DELETE FROM Question WHERE ID = ${id} RETURNING ID", req.params)
```

File: helloCampusService.js (hunk 1 above)
Comment (TheBguy87) [reply helloCampusService.js]: oh yeah, forgot about that one!

**Comment by hangrydave** [comment]
good now

**Pull request merged**